core/src/navigation/ContextSwitcher.html

```diff
@@ -33,6 +53,7 @@
         {options}
         {selectedLabel}
         {selectedOption}
+        {selectedNodePath}
```

Comment:
```suggestion
```
The `ContextSwitcherNav` doesn't seem to use any instance of a  `selectedNodePath`, so no need to give it as a prop to begin with.